index.ts

```diff
@@ -358,6 +360,34 @@ export {
   flipImageVertical,
   rotateImageLeft,
   rotateImageRight,
+  // imageManagers
+  updateSeriesManager,
+  updateSeriesManager as updateLarvitarManager, // alias for backward compatibility
```

Comment:
maybe we could add a warning for deprecation

Reply:
how?

Reply:
<img width="737" alt="Screenshot 2024-12-19 003604" src="https://github.com/user-attachments/assets/b56784bf-0fa6-44ba-b35b-d230efc6c588" />